test/acceptance/eview_test.exs

```diff
@@ -334,7 +329,8 @@ defmodule EViewAcceptanceTest do
                    "rules" => [
                      %{
                        "description" => "value is not allowed in enum",
-                       "params" => ["a", "b"],
+                       "params" => %{"values" => ["a", "b"]},
```

Comment:
@AlexKovalevych this looks like a breaking change and commit message is misleading, b/c you actually changed logical part of code too.

Reply:
@AlexKovalevych I'll rollback this PR and deprecate new version on hex.pm, please send fix without breaking changes if you need version bumps.